recipes/hdf4/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
```

Comment:
Yeah, I'm aware of this sort of issue. A more elegant way to do this is just set it before `make check` and not export it. This is kind of an ugly way to look at this (docs will help here :wink:), but you can do something like [this]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh#L7 ) and [this]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh#L79 ). That way it doesn't get used in build process. It is only used in testing. After testing, everything is changed to rpaths so this is not really a problem in production.